package.json

```diff
@@ -31,10 +31,9 @@
     "yup": "^0.28.3"
   },
   "scripts": {
-    "get-contributors": "bin/generate-contributors-json.js",
-    "prestart": "npm run get-contributors",
+    "prestart": "cpy src/data/contributors.dev.json src/data  --rename=contributors.json",
```

Comment:
Copies the dev environment contributors JSON file so the contributors page can be built when running in the development environment.

Reply:
This really is just a simple copy file, but to make it cross platform, I opted for an npm package. webpack config changes could have been an option as well via the [copy-webpack-plugin](https://webpack.js.org/plugins/copy-webpack-plugin/), but I opted to stay out of webpack land.